EIPS/eip-draft-compact-sig.md

```diff
@@ -0,0 +1,180 @@
+---
+eip: 1
```

Comment:
This seems to be invalid, likely 2098 is the right number.

Reply:
Also name the file `eip-2098.md`

Reply:
Done.